apache/rocketmq/v2/service.proto

```diff
@@ -244,6 +244,63 @@ message ChangeInvisibleDurationResponse {
   string receipt_handle = 2;
 }
 
+message PullMessageRequest {
+  Resource group = 1;
+  MessageQueue message_queue = 2;
+  int64 offset = 3;
+  int32 batch_size = 4;
+  FilterExpression filter_expression = 5;
+  google.protobuf.Duration long_polling_timeout = 6;
+}
+
+message PullMessageResponse {
+  oneof content {
+    Status status = 1;
+    Message message = 2;
+    int64 next_offset = 3;
```

Comment:
Do we need delivery_timestamp like ReceiveMessageResponse

Reply:
The `delivery_timestamp` in `ReceiveMessageResponse` aims to perf the latency of gRPC, we could add it in the next pull request.